commercetools-models/src/main/java/io/sphere/sdk/extensions/AWSLambdaDestination.java

```diff
@@ -0,0 +1,18 @@
+package io.sphere.sdk.extensions;
+
+import com.fasterxml.jackson.annotation.JsonUnwrapped;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import io.sphere.sdk.annotations.FactoryMethod;
+import io.sphere.sdk.annotations.HasBuilder;
+import io.sphere.sdk.subscriptions.AwsCredentials;
+
+@JsonDeserialize(as = AWSLambdaDestinationImpl.class)
+@HasBuilder(factoryMethods = @FactoryMethod(parameterNames = {"arn","awsCredentials"}))
+public interface AWSLambdaDestination extends Destination{
+
+    String getArn();
+
+    @JsonUnwrapped
```

Comment:
Nice that we could use this annotation here instead of having to deal with the properties 👍

Reply:
Yep but we should consider including this in code generator, since the only reason i m not using it to generate the impl class here is that it create `JsonCreator`, which if combined with `JsonUnwrapped` causes jackson framework to throw an exception at runtime.